x/header/header.go

```diff
@@ -0,0 +1,75 @@
+package header
+
+import "net/textproto"
+
+const (
```

Comment:
Do we need all of these values? I think we're only using one or two of those constants. When in doubt, leave it out :)

Reply:
I think it would be great to have those constants and I added the most common headers. I can go with a couple of them.